### PR TITLE
[CARBONDATA-1512] Fixed bug for failing of concurrently executing sql queries in presto

### PR DESCRIPTION
**Pull request by geetikagupta16**
Resolved concurrent query execution failure

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/180/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/305/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/936/

File: integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java [hunk 1]
```diff
@@ -224,15 +225,16 @@ public CarbonTable getTable(SchemaTableName schemaTableName) {
    * is called, it clears this.tableList and populate the list by reading the files.
    */
   private void updateSchemaTables() {
-    // update logic determine later
+// update logic determine later
```
Comment (jackylk) [inline integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java]: incorrect indentation

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/197/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/949/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/369/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/245/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/997/

**Comment by jackylk** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/512/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1139/

**Comment by jackylk** [comment]
LGTM

**Pull request closed without merging**